db/seeds/npq_contracts/fake-2021.csv

```diff
@@ -0,0 +1,78 @@
+Education Development Trust,2021,NPQ for Executive Leadership,1025,925,37
+Education Development Trust,2021,NPQ Leading Literacy,1026,926,38
+Education Development Trust,2021,NPQ Early Years Leadership,1027,927,39
+School-Led Network,2021,NPQ Leading Teaching ,1028,928,40
+School-Led Network,2021,NPQ Leading Behaviour and Culture,1029,929,41
+School-Led Network,2021,NPQ Leading Teacher Development,1030,930,42
+School-Led Network,2021,NPQ for Senior Leadership,1031,931,43
+School-Led Network,2021,NPQ for Headship,1032,932,44
+School-Led Network,2021,NPQ for Executive Leadership,1033,933,45
+School-Led Network,2021,NPQ Leading Literacy,1034,934,46
+School-Led Network,2021,NPQ Early Years Leadership,1035,935,47
+Leadership Learning South East,2021,NPQ Leading Teaching ,1036,936,48
+Leadership Learning South East,2021,NPQ Leading Behaviour and Culture,1037,937,49
+Leadership Learning South East,2021,NPQ Leading Teacher Development,1038,938,50
+Leadership Learning South East,2021,NPQ for Senior Leadership,1039,939,51
+Leadership Learning South East,2021,NPQ for Headship,1040,940,52
+Leadership Learning South East,2021,NPQ for Executive Leadership,1041,941,53
+Leadership Learning South East,2021,NPQ Leading Literacy,1042,942,54
+Leadership Learning South East,2021,NPQ Early Years Leadership,1043,943,55
+Teach First,2021,NPQ Leading Teaching ,1044,944,56
+Teach First,2021,NPQ Leading Behaviour and Culture,1045,945,57
+Teach First,2021,NPQ Leading Teacher Development,1046,946,58
+Teach First,2021,NPQ for Senior Leadership,1047,947,59
+Teach First,2021,NPQ for Headship,1048,948,60
+Teach First,2021,NPQ for Executive Leadership,1049,949,61
+Teach First,2021,NPQ Leading Literacy,1050,950,62
+Teach First,2021,NPQ Early Years Leadership,1051,951,63
+Teacher Development Trust,2021,NPQ Leading Teaching ,1052,952,64
+Teacher Development Trust,2021,NPQ Leading Behaviour and Culture,1053,953,65
+Teacher Development Trust,2021,NPQ Leading Teacher Development,1054,954,66
+Teacher Development Trust,2021,NPQ for Senior Leadership,1055,955,67
+Teacher Development Trust,2021,NPQ for Headship,1056,956,68
+Teacher Development Trust,2021,NPQ for Executive Leadership,1057,957,69
+Teacher Development Trust,2021,NPQ Leading Literacy,1058,958,70
+Teacher Development Trust,2021,NPQ Early Years Leadership,1059,959,71
+UCL Institute of Education,2021,NPQ Leading Teaching ,1060,960,72
+UCL Institute of Education,2021,NPQ Leading Behaviour and Culture,1061,961,73
+UCL Institute of Education,2021,NPQ Leading Teacher Development,1062,962,74
+UCL Institute of Education,2021,NPQ for Senior Leadership,1063,963,75
+UCL Institute of Education,2021,NPQ for Headship,1064,964,76
+UCL Institute of Education,2021,NPQ for Executive Leadership,1065,965,77
+UCL Institute of Education,2021,NPQ Leading Literacy,1066,966,78
+UCL Institute of Education,2021,NPQ Early Years Leadership,1067,967,79
+Ambition Institute,2021,The Early Headship Coaching Offer,300,800,0

```

Comment:
Could you add the following course name to the npq courses locale as it's missing

Reply:
are you referring to these? https://github.com/DFE-Digital/early-careers-framework/blob/develop/config/locales/en.yml#L393 I don't see these been used anywhere and is probably dead code?

Reply:
Sorry meant https://github.com/DFE-Digital/early-careers-framework/blob/develop/config/locales/en.yml#L385 when I seeded my local env and tried accessing payment breakdowns for NPQ I got multiple missing translations, eg:`translation missing: en.npq_courses.npq-early-headship-coaching-offer`

Reply:
nice spot, i'll add those now